### PR TITLE
Create NEWFILE

### PR DESCRIPTION
**Pull request by PromoFaux**

### CODE REVIEW

File: NEWFILE [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (PromoFaux) [inline NEWFILE]: Starting a multi part review

Comment (PromoFaux) [reply NEWFILE]: I also think theis

Comment (PromoFaux) [reply NEWFILE]: It really should say that though.

Comment (PromoFaux) [reply NEWFILE]: test

Comment (PromoFaux) [inline NEWFILE]: Test

Comment (PromoFaux) [inline NEWFILE]: Comment

Comment (PromoFaux) [inline NEWFILE:1]: Here is a review

Comment (PromoFaux) [reply NEWFILE:1]: There are many reviews like it

Comment (PromoFaux) [reply NEWFILE:1]: But this one is mine

**Review by PromoFaux — COMMENTED** [review]
I have done a comment review

File: NEWFILE (hunk 1 above)
Comment (PromoFaux) [reply NEWFILE:1]: > this review has a quote in it

quote

**Review by PromoFaux — COMMENTED** [review]
>This has a quote in it.

test

**Comment by DL6ER** [comment]
Closing this to be able to reopen it as my own PR

**Pull request closed without merging**